pkg/local_object_storage/writecache/flush.go

```diff
@@ -287,7 +285,7 @@ func (c *cache) flushBatch(addrs []oid.Address, ignoreErrors bool) error {
 		defer elapsed(c.metrics.AddWCFlushBatchDuration)()
 	}
 
-	objs := make([]blobstor.PutBatchPrm, 0, len(addrs))
+	objs := make(map[oid.Address][]byte, len(addrs))
 	for _, addr := range addrs {
 		obj, data, err := c.getObject(addr)
```

Comment:
If we don't need `obj`, we can avoid decoding it as well. Just `GetBytes()` and go.